js/default.js

```diff
@@ -115,24 +102,15 @@ window.addEventListener('load', function () {
     dragHandles[i].addEventListener('mousemove', function (e) {
       if (isMouseDown) {
         isDragging = true
-        if (!pauseRecord){
-          mouseLocation(e)
-          recordX = mouseX
-          recordY = mouseY
-          pauseRecord = true
-        }
+        distance += Math.abs(e.movementX) + Math.abs(e.movementY)
```

Comment:
Sorry to interrupt here but as soon I see this the deep in my mind explodes with the need of two squares and sqrt(). 

E.g.: a single movement of 5px by *x* and 5px by *y* has a distance of nearly 7, while you'd calculate it as 10.
For this case it won't probably matter much but I just wanted to point it out.

Reply:
That's the way I had it when I initiated the pull request. When @PalmerAL suggested the use of movementX and movementY, they gave an example for calculating distance:

`distance += Math.abs(e.movementX) + Math.abs(e.movementY);`

I figured that was deliberate, for simplicity or to avoid calculating a lot of square roots. But if we decide that the mathematically rigorous way of doing it is preferable, I can do that!

Reply:
Whatever way you want to do it is fine. I don't think we actually need the accuracy here, and I figured we could simplify this a bit by just estimating, but if you'd prefer to actually calculate the distance that'd be fine too.

Reply:
I also think the sum of absolute values is fine here. When a click should be canceled, the distance should be much larger than the threshold anyways. With the sum of absolute values, we avoid having to compute square roots on the mousemove event, which I think is worth it.

Reply:
OK, I'll go ahead and merge this.